openlibrary/templates/site/footer.html

```diff
@@ -17,7 +17,7 @@
         <!-- Must be loaded after jquery -->
         <script src="//archive.org/includes/analytics.js?v=0c15bbe" type="text/javascript"></script>
         <script>
-        <!-- anonymized analytics -->
+        //anonymized analytics
```

Comment:
this doesn't look right to me,  this is html not js?

Reply:
Actually the comment was in a script tag, therefore it was showing an error in the console

Reply:
Oh, of course, I scanned that way too quickly. Good fix!